### PR TITLE
Fixing flaky e2e tests

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Logging out and logging back in with cypress is very annoying, I don't think it's working like I think it's working. So doing other things so we don't have to do that.

#### Ticket Link
- none

### CODE REVIEW

**Review by lieut-data — APPROVED** [review]

File: tests-e2e/cypress/integration/frontstage/incident_rhs_list_spec.js [hunk 1]
```diff
@@ -1283,11 +1271,13 @@ describe('rhs incident list', () => {
             cy.get('#rhsContainer').should('exist').within(() => {
                 cy.findByText('Your Ongoing Incidents').should('exist');
 
+                cy.get('.scrollbar--view').scrollIntoView();
+
```
Comment (agarciamontoro) [inline tests-e2e/cypress/integration/frontstage/incident_rhs_list_spec.js:1275]: Not sure what we're scrolling into view here.

Comment (cpoile) [reply tests-e2e/cypress/integration/frontstage/incident_rhs_list_spec.js:1275]: Yah, it's a little weird, but it works -- cypress scrolls the rhs list to the top. /shrug

Comment (agarciamontoro) [reply tests-e2e/cypress/integration/frontstage/incident_rhs_list_spec.js:1275]: Cypress dark magic :shrug:

**Review by agarciamontoro — APPROVED** [review]
:tada:

**Pull request merged**